api/TrackEvent.json

```diff
@@ -48,6 +48,55 @@
           "deprecated": false
         }
       },
+      "TrackEvent": {
+        "__compat": {
+          "description": "<code>TrackEvent()</code> constructor",
+          "spec_url": "https://html.spec.whatwg.org/multipage/media.html#the-trackevent-interface",
```

Comment:
I have the same doubt as in https://github.com/mdn/browser-compat-data/pull/11594#discussion_r672967240. I'll hold off merging this until that is resolved.